### PR TITLE
Stop storing balsamic bam

### PR DESCRIPTION
**Pull request by Mropat**
## Description
*This PR adds/fixes ...*

### How to prepare for test
- [ ] ssh to hasta.scilifelab.se
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-hermes-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Comment by Mropat** [comment]
@karlnyr @ashwini06 @ivadym please confirm that removing those files is ok and wont break upload to scout/delivery to customer

**Comment by ashwini06** [comment]
> @karlnyr @ashwini06 @ivadym please confirm that removing those files is ok and wont break upload to scout/delivery to customer

Isn't the bam file is used in the scout YAML file? What does the scout IGV use if bam files are removed?

File: cg_hermes/config/balsamic.py [hunk 1]
```diff
@@ -325,8 +297,6 @@
 }
 
 TUMOR_NORMAL_WGS_TAGS = {
-    frozenset({"normal-bam", "bam"}): {"is_mandatory": True},
-    frozenset({"normal-bam-index", "bam"}): {"is_mandatory": True},
     frozenset({"normal-cram", "cram"}): {"is_mandatory": True},
     frozenset({"cram", "normal-cram-index"}): {"is_mandatory": True},
     frozenset({"tumor-bam", "bam"}): {"is_mandatory": True},
```
Comment (ashwini06) [inline cg_hermes/config/balsamic.py:302]:
```suggestion

```
Also please remove `tumor-bam-index` in the next line

**Review by ashwini06 — CHANGES_REQUESTED** [review]
Maybe you forgot to remove tumor_bam and its index from TN_WGS_TAGS

File: cg_hermes/config/balsamic.py (hunk 1 above)
Comment (Mropat) [reply cg_hermes/config/balsamic.py]: done

**Comment by karlnyr** [comment]
I agree with @ashwini06, we do need the bam upon completion, but when we compress bam->cram we should remove the bam. Currently this occurs on a crontab for mip I believe? Perhaps the function to clean bam/cram in this code can be altered to remove the bam: https://github.com/Clinical-Genomics/cg/blob/master/cg/cli/clean.py

**Review by ashwini06 — APPROVED** [review]

**Pull request merged**